RequestManagementSystem/ConfigTemplate.cfg

```diff
@@ -20,19 +24,23 @@ Services
 }
 Agents
 {
+  ##BEGIN RequestExecutingAgent
   RequestExecutingAgent
   {
     PollingTime = 60
-    RequestsPerCycle = 50
-    MinProcess = 1
-    MaxProcess = 8
-    ProcessPoolQueueSize = 25
+    # number of Requests to execute per cycle
+    RequestsPerCycle = 100
+    # minimum number of workers process in the ProcessPool
+    MinProcess = 2
+    # maximum number of workers process in the ProcessPool
+    MaxProcess = 4
```

Comment:
It was 1 to 8 processes, now you put 2 to 4. What's the rationale?

Reply:
That is what was in the code
https://github.com/DIRACGrid/DIRAC/blob/e80e60df88663927cb6b258920a86c1339a7b1f8/RequestManagementSystem/Agent/RequestExecutingAgent.py#L77-L80

Reply:
in fact please change it to be the same. This is the operational trick to avoid the ProcessPool deadlock...

Reply:
Change what to be the same? min = X max = X  X = ?

Reply:
yes, with `X=20` for example

Reply:
Ok, Done. Also added recommendation to comment for MaxProcess